src/gatsby/src/components/search/mobileFilters.js

```diff
@@ -347,16 +359,15 @@ const MobileFilters = ({
                 Show {totalResults} {totalResults > 1 ? "parks" : "park"}
               </Button>
             </div>
-            <div className="col-12 mt8">
-              <Button
-                variant="outlined"
-                onClick={() => {
-                  handleCloseFilter()
```

Comment:
Does handleCloseFilter need to be deleted?

Reply:
There's an update for CM-909. [Figma](https://www.figma.com/file/CoXGPkDLN7HKkt0kPuqWhv/BC-Parks-Website?type=design&node-id=7268-69728&mode=dev)
- Close(X) button is added on the top right, it has the handleCloseFilter function
- Cancel button is removed (replaced with the Close button)
- Clear filter button is added